builds/misc/templates/base-image-publish/base-image-setup-librocksdb.yaml

```diff
@@ -0,0 +1,18 @@
+steps:
+- pwsh: |
+    $artifactLocale = "$(Build.SourcesDirectory)/librocksdb.so"
+    Invoke-WebRequest -Uri $("https://edgebuild.blob.core.windows.net/librocksdb/librocksdb.so.$(arch)") -OutFile $artifactLocale
+
+    $info = '$(serialize_info)' | ConvertFrom-Json
+
+    # Copy librocksdb.so to the same directory where base image is built
+    # iotedge/edge-agent/docker/linux/arm64v8/base
+    foreach ($each in $info)
+    {
+      $filePath = "$(Build.SourcesDirectory)/$($each.Item1.ToString())/docker/$(os)/$(arch)/base/librocksdb.so"
+      Copy-Item $artifactLocale -Destination "$filePath"
+
+      $isSucceeded = $(if ([System.IO.File]::Exists($filePath)) {"Succeeded"} else {"Failed"})
```

Comment:
Why do we need to check this explicitly? Wouldn't the copy step fail otherwise?

Reply:
It may or may not (depending on the return code). I'd rather have it double check here and print out to console.